### PR TITLE
COMMERCE-5768 Fix the way we're getting countries and regions

### PR DESCRIPTION
**Pull request by alecsloan**

### CODE REVIEW

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by ivicac** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-checkout-web/src/main/resources/META-INF/resources/checkout_step/address.jsp [hunk 1]
```diff
@@ -338,28 +338,27 @@ long commerceRegionId = BeanParamUtil.getLong(currentCommerceAddress, request, "
...
-							for (var i = 0; i < regions.length; i++) {
-								if (
-									regions[i].commerceRegionId ===
-									selectedOption.getData('region')
-								) {
-									commerceRegionIdName.value = regions[i].name;
-
-									break;
-								}
+					function setUIOnlyInputRegionName(regions) {
```
Comment (ivicac) [inline modules/apps/commerce/commerce-checkout-web/src/main/resources/META-INF/resources/checkout_step/address.jsp:341]: @alecsloan I am not sure about this method name

Comment (ivicac) [reply modules/apps/commerce/commerce-checkout-web/src/main/resources/META-INF/resources/checkout_step/address.jsp:341]: Can we call it only setRegionName?

**Review by ivicac — CHANGES_REQUESTED** [review]

**Review by ivicac — APPROVED** [review]

**Comment by alecsloan** [comment]
Hmm, I force pushed a rebase and just saw you tried to forward 2 hours ago. Lets see what happens 🤔

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:sf

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:forward

**Comment by ivicac** [comment]
@alecsloan can you please rebase this PR?

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by alecsloan** [comment]
@ivicac just confirming everthing in here was just a revert to what we had prior. I can change everything to be either all locale or languageId if need be, but both are accepted by the getTitle method.

**Comment by ivicac** [comment]
ci:forward

**Pull request closed without merging**